### PR TITLE
Updating the Homepage

### PR DESCRIPTION
**Pull request by a-laarabi**
#1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,17 +12,16 @@
...
-        </ul>
-    </nav>
+    <a href="#" class="logo">My Logo</a>
+    <ul>
+        <li class="burger"></li>
+        <li class="burger"></li>
+        <li class="burger"></li>
+    </ul>
+
     </header>
```
Comment (muneebulrehman) [inline index.html:22]: The header section/ navbar is not supposed to have a background image. Please update it.
| Figma design  | Your design |
| ------------- | ------------- |
| figma | issue |

File: index.html [hunk 2]
```diff
@@ -32,6 +31,6 @@ <h4>LET’S CONNECT</h4>
             <li><a href="#"><img src="img/github.png" alt="github"></a></li>
             <li><a href="#"><img src="img/angellist.png" alt="angellist"></a></li>
         </ul>
-    </div>
```
Comment (muneebulrehman) [inline index.html:35]: The header body is supposed to have a background image. Please update it.
| Figma design  | Your design |
| ------------- | ------------- |
| figma | issue |The header section/ navbar is not supposed to have a background image. Please update it.
| Figma design  | Your design |
| ------------- | ------------- |
| figma | issue |

File: style.css [hunk 3]
```diff
@@ -26,19 +31,19 @@ header {
...
+  width: 40px;
+  height: 5px;
+  margin-top: 4px;
+  background-color: #6070ff;
+  border-radius: 5px;
 }
 
-h2 {
+h1 {
   font-family: 'Poppins', sans-serif;
```
Comment (muneebulrehman) [inline style.css:43]: Please import this font family for google fonts. You have used this font but it is not available by default on operating systems.

File: index.html (hunk 2 above)
Comment (muneebulrehman) [inline index.html:35]: There is no margin/ padding on the bottom of this section. Kindly add so, I am sure you will like it. Also, the whole app has a background color, if you can check the Figma template you have chosen, it is `#E5E5E5`.

File: README.md [hunk 4]
```diff
@@ -18,14 +18,10 @@ To get a local copy up and running follow these simple example steps.
 
 ### Setup
 
-### Install
-
 ### Usage
 
 ### Run tests
 
-### Deployment
```
Comment (muneebulrehman) [inline README.md:27]: Please update this section, as requested by previous reviewer.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @a-laarabi , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

_Please don't open a new PR when requesting a rereview. This will not put me in sync with the changes requested by the previous reviewer and this is a bad practice. You should work on the PR on which the changes are requested until the PR has been approved._

 ## Highlights

 ## Required Changes ♻️

### Update the description of the pull request. There is no description of what you have done.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 5]
```diff
@@ -26,22 +33,33 @@ header {
...
-  font-size: 15px;
-  color: #344563;
-  padding: 20px;
+.main-text {
+  background: url('img/bg_Header.png');
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-color: #fff;
+  height: calc(100vh - 72px);
+  border-radius: 0 0 0 100px;
```
Comment (cwaku) [inline style.css:50]: <html>
<body>
<p dir="auto" style="box-sizing: border-box; margin-top: 0px !important; margin-bottom: 16px; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">[OPTIONAL] <br>
On the Figma design, there's extra space under this section that your project doesn't have. Please fix that to make your project even better</p>

Figma design | Your design
-- | --
image |  image



</body>
</html>

**Review by cwaku — APPROVED** [review]
Hi @a-laarabi 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**